src/main/java/emt/gthandler/common/loader/EMT_RecipeAdder.java

```diff
@@ -21,6 +22,15 @@ public static void addFusionCraftingRecipe(ItemStack[] inputs, ItemStack output,
         sFusionCraftingRecipes.addRecipe(true, inputs, new ItemStack[]{output}, null, null, null, aDuration, aEUt, aTier);
     }
 
+    public static void addFusionCraftingRecipe(ItemStack[] inputs, FluidStack[] fluidinputs, ItemStack output, FluidStack[] fluidoutputs, int aDuration, int aEUt, int aTier) {
```

Comment:
Why not ItemStack[] output?

Reply:
Ok I made 
- one generic with all arrays 
- one overload with multiple inputs, one output, no fluids
- one overload with multiple inputs, one fluid input, one output, one fluid output

should be enough i think